R/oauth-flow-auth-code.R

```diff
@@ -201,7 +212,12 @@ oauth_flow_auth_code <- function(client,
   )
   utils::browseURL(user_url)
 
-  if (redirect$localhost) {
+  if (redirect$can_fetch_code) {
+    # Wait a bit to give the user a chance to click through the authorisation
+    # process.
+    sys_sleep(2, "for browser-based authentication", progress = FALSE)
```

Comment:
I'm uncertain how to handle this. I really don't love that we block here, it sounds pretty annoying for e.g. Shiny apps. But I also think it would be OK to punt this to a follow-up PR. What do you think?

Reply:
I don't have strong feelings. We can always make it better later.